packages/shared/src/components/search/SearchPanel/SearchPanelInput.tsx

```diff
@@ -44,7 +46,7 @@ export type SearchPanelInputClassName = {
 export type SearchPanelInputProps = {
   className?: SearchPanelInputClassName;
   valueChanged?: (value: string) => void;
-  inputProps?: InputHTMLAttributes<HTMLInputElement>;
+  inputProps?: Omit<InputHTMLAttributes<HTMLInputElement>, 'placeholder'>;
```

Comment:
why this change?

Reply:
Hmm, seems ok to remove actually since it's not being used

Reply:
@nensidosari Like you said, it's not being used, so omitting it as to not have it show up as an available property, giving the impression that it does something  😅